kura/org.eclipse.kura.wire.component.provider/src/main/java/org/eclipse/kura/internal/wire/subscriber/CloudSubscriberOptions.java

```diff
@@ -66,8 +66,7 @@
     int getSubscribingQos() {
         int subscribingQos = DEFAULT_QOS;
         final Object qos = this.properties.get(CONF_QOS);
```

Comment:
agreed.
I'll do that after the completion of another task I'm working on. I'll rebase ma last commit accordingly.